src/NLog/Config/LoggingConfigurationFileLoader.cs

```diff
@@ -51,23 +52,23 @@ namespace NLog.Config
     /// </summary>
     internal class LoggingConfigurationFileLoader : ILoggingConfigurationLoader
     {
-        private static readonly FileWrapper DefaultFileWrapper = new FileWrapper();
-        private readonly IFile _file;
+        private static readonly AppEnvironmentWrapper DefaultFileWrapper = new AppEnvironmentWrapper();
```

Comment:
I think we should rename DefaultFileWrapper  here?

Reply:
Good idea